modules/ezcontent_preview/src/Controller/PreviewView.php

```diff
@@ -42,27 +76,27 @@ public function preview(NodeInterface $node = NULL, $preview_type) {
                 'ezcontent_preview/global',
               ],
             ],
-          );
+          ];
         }
       }
     }
-    
-    // if nothing return as URL
-    // return empty array
-    return array(
+
+    // If nothing return as URL
+    // return empty array.
+    return [
       '#type' => 'markup',
       '#markup' => '',
```

Comment:
Type is not required, it is default.
https://api.drupal.org/api/drupal/developer%21topics%21forms_api_reference.html/7.x#markup